config/initializers/deprecations.rb

```diff
@@ -1 +1,28 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    next if Foreman.in_rake?
+    next unless Rails.env.production?
+    next unless ComputeResource.all.unscoped.map { |cr| cr.class == Foreman::Model::Rackspace }.include? true
```

Comment:
Isn't this equivalent?
```suggestion
    next unless ComputeResource.all.unscoped.any? { |cr| cr.class == Foreman::Model::Rackspace }
```

Reply:
We can bypass the uninitialized constant issue by checking the `type` field, something like: 
```suggestion
    next unless ComputeResource.all.unscoped.where(type: 'Foreman::Model::Rackspace').any?
```